### PR TITLE
Loadbalancer MVP

### PR DESCRIPTION
**Pull request by furkansahin**

### CODE REVIEW

**Comment by enescakir** [comment]
"Load Balancer" is typically two words, not a single one like "Loadbalancer". Is there a specific reason for using it as a single word?

**Comment by furkansahin** [comment]
nah, this is mere a draft pr. needs a lot of clean-up, name is one of them.

File: prog/vnet/update_load_balancer.rb [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+table inet nat {
+  set neighbor_ips_v4 {
+    type ipv4_addr;
+    flags interval;
+    #{neighbor_ips_v4_line}
+  }
+
+  set neighbor_ips_v6 {
+    type ipv6_addr;
+    flags interval;
```
Comment (fdr) [inline prog/vnet/update_load_balancer.rb:55]: btw why `flags interval`? do we use any ranges (a.k.a. intervals) here?

**Comment by furkansahin** [comment]
moved to #1737

**Pull request closed without merging**